nvme-builtin.h

```diff
@@ -67,6 +67,7 @@ COMMAND_LIST(
 	ENTRY("gen-hostnqn", "Generate NVMeoF host NQN", gen_hostnqn_cmd)
 	ENTRY("dir-receive", "Submit a Directive Receive command, return results", dir_receive)
 	ENTRY("dir-send", "Submit a Directive Send command, return results", dir_send)
+	ENTRY("virtualization-management", "Manage Flexible Resources between Primary and Secondary Controller ", virtual_mgmt)
```

Comment:
That is a bit verbose of a name for a shell utility. Let's call it "virt-mgmt".

Reply:
I agree and will update it